app/views/buyers/applications/_widget.html.erb

```diff
@@ -1,5 +1,4 @@
-<%- cinstances = @account.bought_cinstances -%>
-<%- cinstance = cinstances.first! if cinstances.count == 1 -%>
+<%- cinstance = @account.bought_cinstances.first! if @account.bought_cinstances_count == 1 -%>
```

Comment:
I wonder why not `<%- cinstance = @account.bought_cinstances.first -%>` 🤔 because I guess that internally the DB should use the index we have on cinstances user_account_id and it would quickly see that it doesn't have any 🤔

Reply:
We should ask @macejmic 
https://github.com/3scale/system/blame/master/app/views/buyers/applications/_widget.html.erb

Reply:
`<%- cinstance = @account.bought_cinstances.first -%>` looks good to me.